database/mssql/scripts/dbscripts/PSP_PIMS_S15_01/Alter Up/02_PSP_PIMS_S15_00_Alter.sql

```diff
@@ -484,18 +484,6 @@ GO
 IF @@ERROR <> 0 SET NOEXEC ON
 GO
 
--- Create foreign key constraint PIM_PIDSRT_PIM_PRPRTY_FK
```

Comment:
Aqua is trying to add a foreign key constraint here to a code table that has no values. This will always fail as long as there is at least one PIMS_PROPERTY row with a DATA_SOURCE_TYPE_CODE.

Temporarily solved by moving this add statement until after the code table is populated.